### PR TITLE
Application note

### LINKED ISSUE #42: Implement Application Note resource page
Examples:

- https://www.moleculardevices.com/en/assets/app-note/br/assess-virus-neutralization-with-rapid-hts-friendly-assay
- https://main--moleculardevices--hlxsites.hlx.page/en/assets/app-note/br/assess-virus-neutralization-with-rapid-hts-friendly-assay

Tasks:

- [x] Overall content styling (might be mostly ok already)
- [x] PDF button styles incl. icon
- [x] Image alignment, width, margins
- [x] Image caption styling
- [x] Table caption styling

### PR DESCRIPTION
**Pull request by lydiapuric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #42
Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/app-note/br/assess-virus-neutralization-with-rapid-hts-friendly-assay
- After: https://application_note--moleculardevices--hlxsites.hlx.page/en/assets/app-note/br/assess-virus-neutralization-with-rapid-hts-friendly-assay

and 

- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/app-note/br/measure-intrinsic-tryptophan-fluorescence-on-spectramax-id3-microplate-reader
- After: https://application_note--moleculardevices--hlxsites.hlx.page/en/assets/app-note/br/measure-intrinsic-tryptophan-fluorescence-on-spectramax-id3-microplate-reader

Features implemented
- Enable template-specific features
- Overall content styling for application note page
- PDF button styles incl. icon
 - Image alignment, width, margins
 - Captions:
   - Image caption styling
   - Table caption styling
   - Supports multiple lines of caption

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -168,6 +169,22 @@ strong {
...
+em {
+  font-style: italic;
+}
+
+em strong {
+  font-family: var(--ff-proxima-regular);
+  font-weight: 700;
+}
+
+.text-caption {
```
Comment (mhaack) [inline styles/styles.css:181]: Please prefix with `main`
```suggestion
main .text-caption {
```

File: styles/styles.css [hunk 2]
```diff
@@ -297,6 +318,13 @@ main img, main video {
   height: auto;
 }
 
+main img {
+  max-width: 100%;
+  display: block;
+  margin: 0 auto;
```
Comment (mhaack) [inline styles/styles.css:324]: Not sure if this is other implications.

File: styles/styles.css [hunk 3]
```diff
@@ -297,6 +318,13 @@ main img, main video {
...
+  border: 0;
```
Comment (mhaack) [inline styles/styles.css:325]: Isn't this default anyway?

Comment (lydiapuric) [reply styles/styles.css:325]: checking. otherwise i add it into an application-note specific css

Comment (lydiapuric) [reply styles/styles.css:325]: I needed to add following css to center the image. I will add this definition on a template specific css.
main img {
    display: block;
    margin: 0 auto;

File: styles/styles.css (hunk 2 above)
Comment (lydiapuric) [reply styles/styles.css]: as I moved to the template specific css, the implications are now better controlled.

**Review by mhaack — APPROVED** [review]

File: styles/styles.css [hunk 4]
```diff
@@ -314,11 +335,15 @@ main .section>div {
...
     --body-font-size-m: 14px;
+    --body-font-size-s: 13px;
+    --body-font-size-xs: 12px;
   }
 
   a.button:any-link, button {
     font-size: var(--body-font-size-xs);
     padding: 8px 12px;
+    width: 100%;
+    max-width: 200px;
```
Comment (andreituicu) [inline styles/styles.css:346]: Cool :) Looks like I could remove some CSS I added for the carousel. 👍

**Review by andreituicu — APPROVED** [review]

**Review by jose-correia — APPROVED** [review]

**Pull request merged**